jsonschema-core.xml

```diff
@@ -2337,15 +2339,18 @@
                         </t>
                         <t>
                             An array instance is valid against "contains" if at least one of
-                            its elements is valid against the given schema. The subschema MUST be
-                            applied to every array element even after the first match has
```

Comment:
I just noticed these lines got lost inadvertently -- they were added (because unevaluatedItems now considers results from contains) after my original PR. I'll fix.